pkg/pods/pods.go

```diff
@@ -278,7 +303,7 @@ func (obj *Pods) Exec(options ExecOptions) (*ExecResult, error) {
 
 // AddEphemeralContainer adds an ephemeral container to a running pod. The Pod is identified by name and namespace.
 // The container is described by options
-func (obj *Pods) AddEphemeralContainer(name, namespace string, options ContainerOptions) error {
+func (obj *Pods) AddEphemeralContainer(name, namespace string, options EphemeralContainerOptions) error {
```

Comment:
API-breaking change for clients. Since it's early in the process for ephemeral containers, we should be fine.